x-pack/plugins/secops/public/store/local/timeline/model.ts

```diff
@@ -7,25 +7,40 @@
 import { Range } from '../../../components/timeline/body/column_headers/range_picker/ranges';
 import { Sort } from '../../../components/timeline/body/sort';
 import { DataProvider } from '../../../components/timeline/data_providers/data_provider';
-import { ECS } from '../../../components/timeline/ecs';
-import { mockECSData } from '../../../mock/mock_ecs';
 
+export const DEFAULT_PAGE_COUNT = 2; // TODO: Figure out why Eui Pager will not render unless this is a minimum of 2 pages.
```

Comment:
A relatively _probable_ explanation as to why `EuiTablePagination` chose the sentinel value `2`, (when `EuiTablePagination` is pased `2` as value of the `pageCount` `prop`), is `2` is the minimum threshold where it makes sense to show the pager, because (psudocode):

```
switch(pageCount)
0, // no data, so no need to render a pager 👎 
1, // only one page of data, so no need to render a pager 👎 
2, // the user can now toggle between page 1 and 2, render a pager 👍
default: render a pager 👍  
```

Right now, the test passes because the `store` initializes with the value `2`. If all of that makes sense, we can (later) remove this `TODO`, change the default value in the `store` to `0`, and update the test so that `2` is explicitly passed down to force a render of the pager. 😉